module5/JoinsAndGroupBy.txt

```diff
@@ -0,0 +1,109 @@
+psq joins 
+
```

Comment:
Can you add emp, dept columns so which will make easy to understand the joins results.